src/Pos/Block/Arbitrary.hs

```diff
@@ -220,8 +219,7 @@ newtype BlockHeaderList ssc = BHL
     } deriving (Eq)
 
 instance T.BiSsc ssc => Show (BlockHeaderList ssc) where
-    show =
-        concat . intersperse "\n" . map (formatToString build) . uncurry zip . getHeaderList
+    show = intercalate "\n" . map (formatToString build) . uncurry zip . getHeaderList
```

Comment:
Perhaps you used `unlines` from universum? Where are different kinds of `unlines`, universum's one is `[Text] -> Text`. You can import required one from `Prelude`